examples/dynamic.c

```diff
@@ -31,6 +33,10 @@
 #include <unistd.h>
 #include <time.h>
 #include <sys/param.h>
+#include <pmix_config.h>
+#ifdef HAVE_NETDB_H
+#include <netdb.h>
+#endif
```

Comment:
I see - let's just replace that MAXHOSTNAMELEN with some arbitrary number (e.g., 1024) so we remove the issue. It's unimportant in terms of the example we are trying to provide.